src/Http/CookieFactory.php

```diff
@@ -72,45 +71,34 @@ public function __construct(Config $config)
      * This method returns a cookie instance for use with the Set-Cookie HTTP header.
      * It will be pre-configured according to Flarum's base URL and protocol.
      *
-     * @param  string  $name
-     * @param  string  $value
-     * @param  int     $maxAge
-     * @return \Dflydev\FigCookies\SetCookie
+     * @param  string       $name
```

Comment:
Why not just use the question mark before the string / int?

Reply:
phpdoc doesn't allow for conditional type hinting, it is always rendered as constants